hadoop-hdds/common/pom.xml

```diff
@@ -85,6 +85,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>javax.annotation</groupId>
```

Comment:
Do we still need this import?

Reply:
Thanks @aswinshakil for the review.

The answer is yes and no. :)

Yes, because `javax.annotation-api` has a different set of annotations, e.g.: `javax.annotation.PostConstruct`, still used in Ozone.

No, because `hdds-common` itself does not need it.

But it is not strictly related to this change.